extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -143,7 +163,23 @@ export const ManageAssetRows = ({
           const isActionPending = submitStatus === ActionStatus.PENDING;
 
           return (
-            <div className="ManageAssetRows__row" key={canonicalAsset}>
+            <div
```

Comment:
I think it's smart to reuse UI elements ManageAssetsRows for this view, but I think it's a bit confusing to have `ManageAssetsRows` be both a view for editing trustlines and a view for selecting an asset for a payment since they're two pretty dissimilar use cases

My suggestion would be to break out pieces of the UI like `ManageAssetRows__content` and `ManageAssetRows__row` into their own `basics` file with a more generic name. (Check out popup/basics/Forms/index.tsx for an example) Then you could create a new `component` that imports the needed UI elements from that `basics` file with only the logic for dealing with asset selection. Wdyt?

Reply:
thought it might be slightly over-abstracting at first thought, but true the name ManageAssetRows is really a different use case. I'll do some refactoring here 👍

Reply:
added a new component similar to `ManageAssetRows`. I started abstracting the `ManageAssetRows__content` and importing into both but it gets hairy since the row would need access to things like trustlines data for `ManageAssetRows` that the `SelectAssetRows` doesn't use. I think since only used in these two spots this is fine and can re-asses if this UI is used in more spots. Lmk what you think
https://github.com/stellar/freighter/pull/502/commits/dac5c20b3999736d663ae28d41e424bf53729cf0

Reply:
Yeah the repeated styles isn't the best, but if it's too messy to share the styles, that's fine. I think having these 2 components that have clearer purposes is a lot easier to parse 👍 

I would at least make 

```
domain
                    ? domain.replace("https://", "").replace("www.", "")
                    : "Stellar Network"
```

into a helper, though.

Reply:
good idea https://github.com/stellar/freighter/pull/502/commits/3ede4987a0db420640239f6ece2ff7710c100dc8

Reply:
Btw, I think we just need to roll back some of the changes which shouldn't be needed anymore in this file, then we're good to approve! 😄

Reply:
oof ya (I blame it being friday 😄 ), changed back
https://github.com/stellar/freighter/pull/502/commits/ee458a1c69b6913a7c00e1cdc0e1eb185682ecfa